managers/s3.py

```diff
@@ -37,11 +42,79 @@ def createS3Bucket(self, bucketName, bucketPermissions):
         raise S3Error('Unable to create bucket in s3')
 
 
-    def createManifestInS3(self, manifestPath, manifestJSON, s3_bucket: str):
+    def store_pdf_manifest(self, record, bucket_name, requires_login: bool):
```

Comment:
We can default to `requires_login: bool=False` so the other ingest processes don't have to set this parameter.